app/views/hud_reports/_project_filter.haml

```diff
@@ -1,10 +1,11 @@
 - project_types_collection = local_assigns[:project_types_collection] || @filter.available_residential_project_types
 - projects_collection = local_assigns[:projects_collection] || @filter.project_options_for_select(user: current_user)
 %h2 Projects to Include
-- if local_assigns[:alert]
-  %p.alert.alert-info= local_assigns[:alert]
-- else
-  %p.alert.alert-info Please note, the following options are additive.  If you choose a single project and a project type, the report will run for all projects in the project type and the chosen project.  If you choose a data source and project type, the report will run for all projects in the chosen data source and chosen project type.
+%p.alert.alert-info
+  Please note, the following options are additive.  If you choose a single project and a project type, the report will run for all projects in the project type and the chosen project.  If you choose a data source and project type, the report will run for all projects in the chosen data source and chosen project type.
+  - if local_assigns[:alert].present?
+    %br
+    %br= local_assigns[:alert]
```

Comment:
does `%br= ...` turn into `<br>...</br>` or is it smart enough to turn it into `<br /> ...`?

Reply:
Not it just turns it into `<br>`. It looks like this, I just thought there should be some separation between the two messages 
<img width="767" alt="Screen Shot 2022-05-11 at 10 58 08 AM" src="https://user-images.githubusercontent.com/5333982/167881479-5b2c4d61-d51c-44f1-b5c5-ca48f09a11ee.png">